egs2/misp2021/asr1/run.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+
+train_set=train_far 
+valid_set=dev_far
+test_sets=dev_far #test_far
+
+asr_config=conf/tuning/train_asr_conformer.yaml
+lm_config=conf/train_lm.yaml
+inference_config=conf/decode_asr.yaml
+
+use_lm=true
+use_word_lm=false
+
+
+./asr.sh                                   \
+    --lang zh \
+    --audio_format wav                     \
+    --nlsyms_txt data/nlsyms.txt           \
+    --ngpu 3                               \
+    --token_type char                      \
+    --feats_type fbank_pitch               \
```

Comment:
is it possible that we stick to `raw` features for espnet2 (it's ok to use fbank_pitch, I'm just curious as we do not want to have dependencies on Kaldi for espnet2)